READMEs/consume-flow.md

```diff
@@ -0,0 +1,172 @@
+<!--
+Copyright 2022 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: Marketplace Flow
+
+This quickstart describes a batteries-included flow including using off-chain services for metadata (Aquarius) and consuming datasets (Provider).
```

Comment:
Don't need to say "batteries-included". 

E.g. new text: "This quickstart describes how data is consumed / downloaded, including metadata".